Composer/packages/client/src/Onboarding/onboarding.tsx

```diff
@@ -0,0 +1,196 @@
+            )}
+          </div>
+        ),
+        headline: formatMessage('Trigger'),
+      };
+
+    case 'userInput':
+      return {
+        children: (
+          <div>
+            {formatMessage('You can define and manage ')}
+            <b>{formatMessage('intents ')}</b>
+            {formatMessage(
+              'here. Each intent describes a particular user intention through utterances (i.e. user says). '
+            )}
+            <b>{formatMessage('Intents are often triggers of your bot.')}</b>
+          </div>
+        ),
+        headline: formatMessage('User input'),
+      };
+
+    case 'actions':
+      return {
+        children: (
+          <div>
+            {formatMessage('Actions define ')}
+            <b>{formatMessage('how the bot responds ')}</b>
+            {formatMessage('to a certain trigger.')}
+          </div>
+        ),
+        headline: formatMessage('Actions'),
+      };
+
+    case 'botResponses':
+      return {
+        children: (
+          <div>
+            {formatMessage(
+              'You can manage all bot responses here. Make good use of the templates to create sophisticated response logic based on your own needs.'
+            )}
+          </div>
+        ),
+        headline: formatMessage('Bot responses'),
+      };
+
+    case 'welcomeMessage':
+      return {
+        children: (
+          <div>
+            <div>
+              {formatMessage('The welcome message is triggered by the ')}
+              <i>{formatMessage('ConversationUpdate ')}</i>
+              {formatMessage('event. You may customize or add a new one. To do this:')}
+            </div>
+            <ol>
+              <li>
+                {formatMessage('Create a new or go to the trigger: ')}
+                <i>{formatMessage('ConversationUpdate')}</i>
+              </li>
+              <li>{formatMessage('Select/add action: send an Activity')}</li>
+              <li>{formatMessage('Edit the message in the right pane')}</li>
+            </ol>
+          </div>
+        ),
+        headline: formatMessage('Add welcome message'),
+      };
+
+    case 'intentTrigger':
+      return {
+        children: (
+          <div>
+            {formatMessage(
+              'Click on the + and select intent as the trigger type. Follow the wizard to define the intent and other trigger settings. Then add actions in the visual editor.'
+            )}
+          </div>
+        ),
+        headline: formatMessage('Add an intent trigger'),
+      };
+
+    case 'startBot':
+      return {
+        children: (
+          <div>
+            {formatMessage('Click on the ')}
+            <b>{formatMessage('Start Bot ')}</b>
+            {formatMessage('button. Then you can choose to ')}
+            <b>{formatMessage('Test in Bot Framework Emulator. ')}</b>
+            {formatMessage('This will open your Emulator app.')}
```

Comment:
I think we should adjust the copy to say:

> This will open your Emulator application. If you don't yet have the Bot Framework Emulator installed, you can download it here (link to https://github.com/microsoft/BotFramework-Emulator/releases/latest)